### ISSUE TITLE
Name resultsviewer from flowrunner on tablet 

### ISSUE THREAD

**Issue by SaraMagits** [exporl/apex#250]
When automatically saving a resultsfile from the flowrunner, the name is defined only by the elements in the flowrunner (e.g., subject_code or study_code) but the filename of the experiment is not saved. This in contrast to the desktop version of APEX where the resultsfile is named filenameExperiment_subjectCode_studyCode.apr

**Comment by SaraMagits** [comment]
This is incorrect, also on desktop the experiment file name has to be defined in the flowrunner for it to be included in the results filename. There is a problem when using the Data Manager, the results file is then named 'ExperimentFileName_DateTimeStamp.apr'. It seems the results file name defined in the flowrunner is overlooked or overruled.

**Comment by SaraMagits** [comment]
I believe this can be closed. @wardd64 , can you confirm?

**Comment by wardd64** [comment]
> I believe this can be closed. @wardd64 , can you confirm?

Yes, this may be closed, the solution has been merged already:
https://exporl.med.kuleuven.be/gerrit/gitweb?p=apex.git;a=commit;h=7474db466b326587f82e82353f0b41b8edf38205
